tests/default.nix

```diff
@@ -48,6 +50,10 @@ in {
   kibana7 = callTest ./kibana.nix { version = "7"; };
   k3s = callTest ./k3s {};
   kvm_host = callTest ./kvm_host.nix {};
```

Comment:
Suggestin: rename to kvm_host_jewel-ceph-server-jewel and avoid a version-less alias